modules/microshift-check-cluster-status.adoc

```diff
@@ -6,33 +6,50 @@
 [id="microshift-check-cluster-status_{context}"]
 = Checking the status of a cluster
 
-You can check the status of a {microshift-short} cluster or see active pods by running a simple command. Given in the following procedure are three commands you can use to check cluster status. You can choose to run one, two, or all commands to help you retrieve the information you need to troubleshoot the cluster.
+You can check the status of a {microshift-short} cluster or see active pods. Given in the following procedure are three different commands you can use to check cluster status. You can choose to run one, two, or all commands to help you get the information you need to troubleshoot the cluster.
 
 .Procedure
-* You can check the system status, which returns the cluster status, by running the following command:
+* Check the system status, which returns the cluster status, by running the following command:
 +
 [source,terminal]
 ----
 $ sudo systemctl status microshift
 ----
 +
-If {microshift-short} is failing to start, this command returns the logs from the previous run.
+If {microshift-short} fails to start, this command returns the logs from the previous run.
++
+.Example output
+[source,text]
+----
+● microshift.service - MicroShift
+     Loaded: loaded (/usr/lib/systemd/system/microshift.service; enabled; preset: disabled)
+     Active: active (running) since <day> <date> 12:39:06 UTC; 47min ago
```

Comment:
Not sure if I am reading this incorrectly, but this says `active`. If it fails to start, I think the message would be different.

Reply:
This is healthy output. I can specify that.